.travis.yml

```diff
@@ -14,7 +14,7 @@ script:
 deploy:
   -
     provider: script
-    script: "./.travis/deploy.sh"
+    script: chmod +x .travis/deploy.sh && bash .travis/deploy.sh
```

Comment:
This is weird: AFAIK, git records and preserves permissions. You should make it executable on your machine and it should be executable on any *IX machine